crates/storage_models/src/merchant_account.rs

```diff
@@ -48,7 +48,7 @@ pub struct MerchantAccountNew {
 #[derive(Debug)]
 pub enum MerchantAccountUpdate {
     Update {
-        merchant_id: String,
+        merchant_id: Option<String>,
```

Comment:
Should we allow the update of api_key, merchant_id, publishable key ?

Reply:
merchant_id can't be updated.

api_key and publishable_key can be updated